saleor/webhook/payloads.py

```diff
@@ -93,6 +93,11 @@ def prepare_order_lines_allocations_payload(line):
     return warehouse_id_quantity_allocated_map
 
 
+def _charge_taxes(order_line: OrderLine) -> bool:
+    variant = order_line.variant
+    return False if not variant else variant.product.charge_taxes
```

Comment:
I would say, that here should be None instead of False. With False, external app will not be able to determine if the charge taxes is false because, variant doesn't exist or it doesn't expect charges in taxes.